src/System.Security.Cryptography.X509Certificates/tests/LoadFromFileTests.cs

```diff
@@ -167,7 +167,7 @@ public static void TestLoadSignedFile()
                 byte[] expectedThumbprint = expectedThumbprintHash.HexToByteArray();
                 byte[] actualThumbprint = c.GetCertHash();
                 Assert.Equal(expectedThumbprint, actualThumbprint);
-#if netstandard17
+#if netcoreapp11
```

Comment:
Why did you change all of these? I see a default TargetGroup build in the .builds file, which according to the csproj makes it netstandard1.7.

The "netcoreapp11" \#define is for testing API that is present in .NET Core, but not in .NET Standard. So really only the ephemeral import tests should be conditionally-netcoreapp11; all of the tests you're adding (and have modified) should (still) be conditionally-netstandard17.